rct229/rulesets/ashrae9012019/data_fns/table_G3_1_1_2_fns.py

```diff
@@ -0,0 +1,26 @@
+from rct229.rulesets.ashrae9012019.data import data
+from rct229.rulesets.ashrae9012019.data_fns.table_utils import find_osstd_table_entry
+
+
+def table_g3_1_2_lookup(building_area_type: str) -> dict[str:str]:
+    """
+
+    Parameters
+    ----------
+    building_area_type: str
+        building area type of the service water heating system
```

Comment:
Specify that this `building_area_type` is ServiceWaterHeatingSpaceOptions2019ASHRAE901

Reply:
Addressed.